### DISCUSSION TITLE
Using docker CE in prod

### DISCUSSION THREAD

**Discussion by satyaki1008** [BretFisher/ama#159] [Q&A]
Can I use docker CE in prod setup?
If yes what should I lookout for before using it in prod leaving aside the already mentioned security aspects in your blog:https://www.bretfisher.com/dockercon18/

**Comment by BretFisher** [comment]
Yes, most docker users and nearly all Kubernetes users are using the CE (free open source edition).

You can go through my resources here: https://www.bretfisher.com/docker

Watch the DockerCon videos, which are specific to production topics.

Good luck with your container projects!
